src/index.html

```diff
@@ -1,12 +1,44 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page__body">
+    <div class="page__calendar calendar calendar--start-day--wed calendar--month-length--28">
```

Comment:
To avoid error `Tag can't have more than one block` pay attention to your classes. 
Remove extra class `page__calendar`
```suggestion
    <div class="calendar calendar--start-day--wed calendar--month-length--28">
```

Reply:
Probably I'm doing smth wrong. Linter shows errors then classes are divided into lines. 
<img width="602" alt="Screenshot 2023-03-01 at 13 05 34" src="https://user-images.githubusercontent.com/9693794/222160090-d947668a-9765-4fe2-8f8f-a8d27093ecc2.png">

<img width="565" alt="Screenshot 2023-02-28 at 16 28 18" src="https://user-images.githubusercontent.com/9693794/222160071-2f5576bb-c979-4287-8833-ec54ec7459f8.png">